requirements.txt

```diff
@@ -1,62 +1,63 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file requirements.txt requirements.in
-#
-amqp==2.2.2               # via kombu
-attrs==17.4.0             # via pytest
-autopep8==1.3.5           # via django-silk
-babel==2.6.0              # via flower
-billiard==3.5.0.3         # via celery
-bleach==2.1.3
-cachetools==3.1.0         # via google-auth
-celery==4.1.0
-certifi==2018.4.16        # via requests
-chardet==3.0.4            # via requests
-click==6.7                # via pip-tools
+-i https://pypi.org/simple
```

Comment:
You can remove `requirements.txt` if we have a pipfile. And let's replace `pip install -r requirements.txt` from the [Dockerfile](https://github.com/hoover/snoop2/blob/96d1e72b2cece28fdd668f784d9c1c0131aaf47e/Dockerfile#L23) with `pipenv install`:

https://github.com/hoover/reup/blob/37c947ec8f51db3b36f9cf5f8313b269c7acc06b/Dockerfile#L10-L12